### ISSUE TITLE
Build alternate version of homepage with a video embedded on it

### ISSUE THREAD

**Issue by alanmoo** [mozilla/donate.mozilla.org#1816]
Shouldn't be entirely blocked by design

**Comment by gideonthomas** [comment]
Adding a comment from another ticket from @cadecairos re: this:

> ensure it's click-to-load so that overal page load times are minimally impacted.

**Comment by gideonthomas** [comment]
Closing as this is going to be part of a single effort outlined in #1814
